blimflix/client/app/core/sidebar.scss

```diff
@@ -1,75 +1,78 @@
 .sidebar-wrapper {
-height: 100%;
-background-color: #222;
-position: fixed;
-left: 0;
-top: 50px;
-padding: 0;
-padding-top: 70px;
-border-right: 2px solid black;
-}
-
-.profile-wrapper {
-height: 50px;
-color: #FFFFFF;
-}
-
-.row {
-    color: #FFFFFF;
-}
+  height: 100%;
+  background-color: #222;
+  position: fixed;
+  left: 0;
+  top: 50px;
+  padding: 0;
+  padding-top: 70px;
+  border-right: 2px solid black;
 
-.row img {
-    width: 100%;
-    height: 100%;
 }
 
-.row h4 {
-    font-weight: bold;
-}
+.profile-wrapper {
+  height: 50px;
+  color: #FFFFFF;
 
-.row h4, .row h5 {
-    padding: 0;
-    margin:  0;
-    padding-top: 5px;
-    padding-left: 10px
 }
 
-.col-centered{
-    float: none;
-    margin: 0 auto;
+.row {
```

Comment:
Very good changes, but this rule is too generic.
In every place that you have a .row with .col-xs-3 child class (.row> .col-xs-3> img)
This is a very common structure
I would recommend to make structures like this
```
.sidebar {
     .row {
         ...
     }
     ...
}
```

At this point this could not make problems, but it could when pages are develop